casper/src/main/resources/ListOps.rho

```diff
@@ -193,6 +193,25 @@ new  ListOps, rs(`rho:registry:insertSigned:secp256k1`), uriOut in {
     }
   } |
 
+  contract ListOps(@"any", @list, predicate, returnCh) = {
```

Comment:
We don't have to mimic scala method naming conventions but this should have a more informative name. This should have a more informative name. Scala and Haskell use `exists`.